data_reports/108_Lehmann_NE_US_soils.Rmd

```diff
@@ -1,30 +1,211 @@
----
-title: "108_Lehmann_NE_US_soils"
-author: "Julie Bielecki"
-date: "3/23/2021"
-output: html_document
----
+# Lehmann NE US soils
+
+The Lehmann NE US soils data set in ISCN3 contains `r nrow(dataset_layer)` layer-level rows and `r nrow(dataset_layer)` profile-level information rows after cleaning for ISCN3.5.
```

Comment:
Because you haven't created `dataset_layer` or `dataset_profile` these `nrow` commands won't run when you transfer them. I like the inclination to use r command here but for this specific application you do actulaly need to write out the number of rows here.